Cargo.toml

```diff
@@ -78,16 +78,15 @@ core-protocol = { path = "transport/protocol", default-features = false }
 core-p2p = { path = "transport/p2p", default-features = false }
 core-strategy = { path = "logic/strategy", default-features = false }
 core-transport = { path = "transport/api", default-features = false }
```

Comment:
Indeed. I've added `hopr-` prefices only to the packages that are in their final position and are fully updated. The remaining packages are placed in a correct directory, e.g. `transport/api`, but should be either refactored, documented or merged with other packages.

Case and point:
1. `logic/path` and `logic/strategy` are too tightly bound to everything, they need to be abstracted away with traits and isolated from the large amount of transport and chain packages.
2. chain packages can be refactored and joined, e.g. `chain_types` could likely be extracted to `hopr-internal-types`, `chain_api` should be dissolved and joined with `chain_actions`,... etc.
3. `core-p2p` needs to be abstracted by a trait interface that is yet to live somewhere...

That's why I've opted for renaming those that are eventually refactored... so that the packages do not intermix.

Reply:
By this logic, hopr-inbox and hopr-keypair should be renamed to just inbox and keypair, as they might need another look.